public/docs/_examples/attribute-directives/dart/lib/highlight_directive.dart

```diff
@@ -0,0 +1,55 @@
+// #docplaster
+// #docregion full
+library attribute_directives.highlight_directive;
+
+import 'package:angular2/angular2.dart';
+
+@Directive(selector: '[my-highlight]', host: const {
+  '(mouseenter)': 'onMouseEnter()',
+  '(mouseleave)': 'onMouseLeave()'
+})
+// #docregion class-1
+class Highlight {
+  // #enddocregion class-1
+// #enddocregion full
+  /*
+// #docregion highlight
+  @Input() myHighlight: string;
+// #enddocregion highlight
+  */
+// #docregion full
+// #docregion class-1
+// #docregion color
+  @Input('my-highlight') String highlightColor;
+// #enddocregion color
+
+  String _defaultColor = 'red';
+  // #enddocregion class-1
+  // #docregion defaultColor
+  @Input() set defaultColor(String colorName) {
```

Comment:
The return value is `null` anyway, but I know that setter are currently quite weird. Just wanted to get this topic some attention.